samples/batch/csharp/program.cs

```diff
@@ -99,22 +100,26 @@ static async Task TranscribeAsync()
                             }
                             completed++;
 
-                            // if the transcription was successfull, check the results
+                            // if the transcription was successful, check the results
                             if (transcription.Status == "Succeeded")
                             {
                                 var resultsUri0 = transcription.ResultsUrls["channel_0"];
-                     
+
                                 WebClient webClient = new WebClient();
 
                                 var filename = Path.GetTempFileName();
                                 webClient.DownloadFile(resultsUri0, filename);
                                 var results0 = File.ReadAllText(filename);
                                 var resultObject0 = JsonConvert.DeserializeObject<RootObject>(results0);
                                 Console.WriteLine(results0);
-                                
+
                                 Console.WriteLine("Transcription succeeded. Results: ");
                                 Console.WriteLine(results0);
                             }
+                            else
+                            {
+                                Console.WriteLine("Transcription failed. Status: {0}", transcription.StatusMessage);
+                            }
```

Comment:
Could be cleaner to have own case for "Failed" case and potentially then we could remove comparisons to (transcription.Status == "Succeeded") from the code and make it more straightforward.